ReSwift/CoreTypes/Store.swift

```diff
@@ -204,4 +213,18 @@ extension Store where State: Equatable {
             }
             _ = subscribe(subscriber, transform: { $0.skipRepeats() })
     }
+
+    open func subscribe<SelectedState: Equatable, S: StoreSubscriber>(
+        _ subscriber: S, transform: ((Subscription<State>) -> Subscription<SelectedState>)?
+        ) where S.StoreSubscriberStateType == SelectedState
+    {
+        let originalSubscription = Subscription<State>()
+
+        var transformedSubscription = transform?(originalSubscription)
+        if subscriptionsAutomaticallySkipRepeats {
+            transformedSubscription = transformedSubscription?.skipRepeats()
```

Comment:
What will happen here now if we call `skipRepeats()` manually on substate with this enabled? Will it cause a double-skip-repeats? (Sort of relates to https://github.com/ReSwift/ReSwift/pull/259 )

Aside: I think the amount of subscription bugs & things we've been dealing with lately are calling for a more substantial refactor of the internals of subscriptions, given the amount of side-effects that are showing up

Reply:
Right now, as I understand it, it will cause a double skip repeats which is the same thing outlined in #259.

Reply:
Alright, just wanted to confirm that. I'm okay with it having the same behaviour as we already do now.